<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDispPlanDatesDialog</class>
 <widget class="QDialog" name="ExportDispPlanDatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Даты запланированных мероприятий</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Подразделение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDbComboBox" name="cmbOrgStructure">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CInDocTableView" name="tblPlanDates">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ошибки при экспорте:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CTableView" name="tblPlanDateErrors">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CDbComboBox</class>
   <extends>QComboBox</extends>
   <header>library/DbComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
